server/controller/recipe.js

```diff
@@ -0,0 +1,62 @@
+import db from '../models/index';
+import { validateRecipes } from '../validators/validator.js';
+import { sendValidationError, serverError, sendSuccess, sendFail } from '../reply/reply';
+
+const Recipes = db.Recipes;
+const log = console.log;
```

Comment:
Instead of doing this, you use a logger that shows the request/response status of your app on the console

Reply:
am using a logger
this is for logging out my output